x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/calculate_rule_source_for_import.ts

```diff
@@ -26,31 +27,44 @@ import { convertRuleToImportToRuleResponse } from './converters/convert_rule_to_
  * @returns The calculated rule_source and immutable fields for the rule
  */
 export const calculateRuleSourceForImport = ({
-  rule,
+  importedRule,
+  currentRule,
   prebuiltRuleAssetsByRuleId,
   isKnownPrebuiltRule,
```

Comment:
nit: It could be simplified by having only `prebuiltRuleAssets: PrebuiltRuleAsset | undefined` matching the rule version instead of  `isKnownPrebuiltRule` and `prebuiltRuleAssetsByRuleId `.